### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@kubesphere.io>

@FeynmanZhou @benjaminhuo @tpiperatgod 
Hi, please take a look at the updates I made, thanks.

### CODE REVIEW

**Review by tpiperatgod — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -111,52 +113,52 @@ func BindingsOutput(ctx *ofctx.OpenFunctionContext, in []byte) ofctx.RetValue {
...
 
-[Here](docs/roadmap.md) you can find OpenFunction's roadmap.
+Learn more about OpenFunction [roadmap](docs/roadmap.md).
 
 ## 🏘️ Community
 
 ### Community Call
 
-Meeting Info: [Zoom](https://us02web.zoom.us/j/89684762679?pwd=U1JNWVdzbElScVFMSEdQQnV0YnR4UT09)
+Meeting venue: [Zoom](https://us02web.zoom.us/j/89684762679?pwd=U1JNWVdzbElScVFMSEdQQnV0YnR4UT09)
```
Comment (FeynmanZhou) [inline README.md:150]: Is `venue` suitable for a virtual meeting?

Comment (benjaminhuo) [reply README.md:150]: Dapr call it: Community Call : https://github.com/dapr/dapr or community meeting https://github.com/dapr/community#community-meetings

Keda call it Meeting Time under Community: https://github.com/kedacore/keda

Comment (Felixnoo) [reply README.md:150]: > Member

Right...I'll change it.

Comment (Felixnoo) [reply README.md:150]: > Dapr call it: Community Call : https://github.com/dapr/dapr or community meeting https://github.com/dapr/community#community-meetings
> 
> Keda call it Meeting Time under Community: https://github.com/kedacore/keda

Sure.

Comment (Felixnoo) [reply README.md]: [Updated](https://github.com/OpenFunction/OpenFunction/pull/195/commits/b9f8a5504b0356b6edfc9e33d1ab833f7467c7a2).

**Review by benjaminhuo — APPROVED** [review]
Nice changes

**Pull request merged**